Shared/Actions/Shots/HitscanShot.gd

```diff
@@ -18,19 +26,23 @@ func initialize(owning_player) -> void:
 	if owning_player.has_node("KinematicBody/CharacterModel"):
 		var character_model_controller = owning_player.get_node("KinematicBody/CharacterModel")
 		var color_scheme = character_model_controller.color_scheme
-		color_name = color_scheme + "_primary_accent"
+		color_name = color_scheme + "_main"
 	
 	# TODO: Check is necessary because server does not do any coloring, we should maybe 
 	# make a client only version for this class
 	if get_node("/root").has_node("ColorManager"):
 		var color_manager = get_node("/root/ColorManager")
 		color_manager.color_object_by_property(color_name, $Visualisation.material_override, "albedo_color")
+		color_manager.color_object_by_property(color_name, $Visualisation.material_override, "emission")
 		color_manager.color_object_by_property(color_name, $HitPoint/FrontParticles.material_override, "emission")
 		color_manager.color_object_by_property(color_name, $HitPoint/BackParticles.material_override, "emission")
 
 	cast_to = Vector3(0,0,-_bullet_range)
 	_owning_player = owning_player
 	add_exception(owning_player.get_body())
+	
+	PostProcess.animate_property("chromatic_ab_strength", 0.5, 0, _max_time)
```

Comment:
This now also triggers those effects if any other character than the player spawns a hitscan shot. I don't think we want this.

Reply:
Don't we? I found the effect to be pretty cool, it makes combat very lively, and it thematically kind of makes sense that the other character shooting causes this "glitch" too. But if the consensus is that we don't want it, I can change it. (@incredibleLeitman @DavidPgl)

Reply:
I mean, if we compare it to other games, your really only get camera shake etc. when something directly concerns you. A random ghost shooting in the corner opposite from the player triggering the shake might really confuse the player.

Reply:
But yeah its hard to say, without testing it during a real game.

Reply:
True, we might want to deactivate it for ghosts. But I think it'd be cool to have at least for the other active player. Should I leave it as is for now and open a 'Needs Testing' issue for it after it's merged?

Reply:
Maybe only do it when the player shoots or gets hit (enemy or ghost)?

Reply:
When the player gets hit, we have the more exaggerated death effect anyways

Reply:
> True, we might want to deactivate it for ghosts. But I think it'd be cool to have at least for the other active player. Should I leave it as is for now and open a 'Needs Testing' issue for it after it's merged?

Maybe extract the triggering to the character base class so that if we decide to disable it for other characters, we can just flip a boolean or something. But yea should be fine with a needs testing issue.

Reply:
Sounds good :+1: